MetadataExtractor/Formats/Png/PngMetadataReader.cs

```diff
@@ -373,7 +373,7 @@ IEnumerable<Directory> ProcessTextChunk(string keyword, byte[] textBytes)
                         yield return ReadTextDirectory(keyword, textBytes, chunkType);
                     }
                 }
-                else if (keyword == "Raw profile type exif" || keyword == "Raw profile type APP1")
+                else if (keyword is "Raw profile type exif" or "Raw profile type APP1")
```

Comment:
Could keyword be a span?

Reply:
Almost. The keyword is materialized here, so no win keeping it UTF-8.
 
<img width="836" alt="Screenshot 2024-01-27 at 1 36 21 AM" src="https://github.com/drewnoakes/metadata-extractor-dotnet/assets/7537/1571a214-199d-4681-bc36-8bb325fba9c7">